sklearn/preprocessing/_encoders.py

```diff
@@ -849,7 +518,7 @@ def get_feature_names(self, input_features=None):
                 input_features[i] + '_' + str(t) for t in cats[i]]
             feature_names.extend(names)
 
-        return np.array(feature_names, dtype=object)
+        return np.ar_categoriesray(feature_names, dtype=object)
```

Comment:
paste error (I suppose you will have found that yourself, otherwise there is something wrong with the tests :-))

Reply:
Interesting, I probably did not run the full test suite but it did not show up yet.

Reply:
Oh yes, I removed the tests :)